app/js/lib/foxbox/recipes.js

```diff
@@ -54,9 +54,8 @@ export default class Recipes {
       'POST',
       { getters: [{ kind: 'ThinkerbellRuleSource' }] }
     )
-    .then((services) => {
       // Mark getters and setters with more friendly names.
-      return services.map((service) => {
+      .then((services) => services.map((service) => {
```

Comment:
About question 2: Yes, this style is enforced by `arrow-body-style`. I think you're right, it doesn't always make this easier to read. I suggest to make it only a warning for now so that we can decide whether to use it or not. I'll also revert the ones that are hard to read. Is that OK?